package.json

```diff
@@ -69,7 +69,7 @@
 		"zh_TW": "繁體中文"
 	},
 	"dependencies": {
-		"@godaddy-wordpress/coblocks-icons": "1.3.0"
+		"@godaddy-wordpress/coblocks-icons": "^1.5.3"
```

Comment:
First observation - this should be `1.6.0` since that is the latest version and already merged on `master`